src/devtools/useAtomsDevtools.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  AtomState,
+  DEV_GET_ATOM_STATE,
+  DEV_SUBSCRIBE_STATE,
+  WRITE_ATOM,
```

Comment:
Please see how `useAtomsSnapshot` is implemented.
We should use DEV_GET_MOUNTED_ATOMS. We should not keep unmounted atoms.
A snapshot should hold all mounted atoms and it shouldn't be mutated after creation.
And, please use RESTORE_ATOMS, not WRITE_ATOM.

Reply:
I get what you mean. but I don't find a solution with using snapshots (as you say). So let me try with atom.toString and push my findings. then let me know your opinion.